app/connectors/CustomsDeclareExportsMovementsConnector.scala

```diff
@@ -33,31 +33,41 @@ import scala.util.{Failure, Success}
 class CustomsDeclareExportsMovementsConnector @Inject()(appConfig: AppConfig, httpClient: HttpClient)(implicit ec: ExecutionContext) {
 
   private val logger = Logger(this.getClass)
-
-  private val CustomsDeclareExportsMovementsUrl = s"${appConfig.customsDeclareExportsMovements}"
-
-  private val movementSubmissionUrl: PartialFunction[JourneyType, String] = {
-    case JourneyType.ARRIVE | JourneyType.DEPART => s"$CustomsDeclareExportsMovementsUrl${appConfig.movementsSubmissionUri}"
-  }
-
   private val JsonHeaders = Seq(HeaderNames.CONTENT_TYPE -> ContentTypes.JSON, HeaderNames.ACCEPT -> ContentTypes.JSON)
 
-  def sendArrivalDeclaration(movementRequest: MovementRequest)(implicit hc: HeaderCarrier): Future[HttpResponse] =
-    postRequest(JourneyType.ARRIVE, movementRequest)
+  @Deprecated()
```

Comment:
Can you add a message what should be used instead?

Reply:
I'll go one step better and just delete them 👍